app/views/user_mailer/new_contacts_digest.html.erb

```diff
@@ -17,4 +17,5 @@ PLEASE NOTE: Guests and hosts have NOT been vetted by the campaign, by<br>
 volunteers, or by anyone else. It is your responsibility to contact<br>
 potential hosts and visitors in advance and decide whether you are<br>
 comfortable and safe staying with them.<br><br>
-You can add, edit or delete your host offers anytime here: <%= link_to "HillaryBNB - Host", edit_hosting_url(@hosting) %>
+You can add, edit or delete your host offers anytime here: <%= link_to "HillaryBNB - Host", edit_hosting_url(@hosting) %><br>
+To stop receiving emails from HillaryBNB, you can <%= link_to "unsubscribe", unsubscribe_url() %>.
```

Comment:
Yeah, I am fixing these. The random token seems like a better idea (the same way `confirm_email` works).